src/main/java/com/WebController/EventController.java

```diff
@@ -65,4 +33,37 @@
         return events;
     }
 
+    @PostMapping (value = "/eventinfo")
```

Comment:
the method is called addEvent(...) but it returns a Patient. This is not intuitive.

Reply:
How about addAttendedEvent()? Or addPatientEvent()?

Reply:
I guess you need to return the patient so that you can use the id as the session on the client side, so that's ok. Maybe a better name for the method would be addPatientEventAssociation()?